objectcomputing/check-ins
# Feature 2190/replace transfer list on email page with member selector

This changes the Email page to use the new MemberSelector component in place of the transfer list for selecting recipients.

To test:
- Log in as an admin
- Go to ADMIN > Email
- Write a bogus email in text format to proceed to step 3: Select Recipients
- Ensure that selecting recipients works correctly

I could not get the emails to actually send due to an internal server error on the back-end, but the list of recipients sent to the server seems to be correct. 

Closes #2190
